modules/text_on_image/text_on_image_237_Harishankar_R_.html

```diff
@@ -0,0 +1,29 @@
+<div id="textOnImageContainer">
```

Comment:
@harishankarr7 Add HTML boilerplate

```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <style>
    <!-- Add your style code in between this -->
    </style>
</head>
<body>
    <!-- Add your HTML code in between this -->
</body>
</html>
```

Reply:
@krushnarout please 🙏 consider this PR as I discovered that we have other modules using this same style/approach.

Our AI system launch would clean that. Using our role based programs, We can also handle an automatic correction. So we can decide to ignore it.

Thank you.

Reply:
Okay thanks, I will merge this PR